package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@gooddollar/gooddapp",
   "version": "0.24.1-3",
   "engines": {
-    "node": "10.x",
+    "node": "10.21.0",
```

Comment:
i remember we had issues with `scrypt` build under node > 10.15
is it working now ? If remove `node_modules` and clean npm cache - will npm install  pass without any issues on 10.21 ?

Reply:
yes